cmd/kube-apiserver/app/server.go

```diff
@@ -160,7 +161,7 @@ cluster's shared state through which all other components interact.`,
 }
 
 // Run runs the specified APIServer.  This should never exit.
-func Run(completeOptions completedServerRunOptions, stopCh <-chan struct{}) error {
+func Run(ctx context.Context, completeOptions completedServerRunOptions, stopCh <-chan struct{}) error {
```

Comment:
I'm all for switching to context, but having both context and stopCh is super misleaing.